### ISSUE TITLE
Error with app.ini

### ISSUE THREAD

**Issue by RafaElZgz** [0xJacky/nginx-ui#66]
My problem is that no matter what I put in the app.ini file, the service ignores it and runs with default values. 

I always get the alert that the service is running in debug mode and the port is always 9000. I haven't done anything special, just run the installation script.

I am using ubuntu se 22.04.

**Comment by 0xJacky** [comment]
Have you restart nginx-ui after modify app.ini?

**Comment by RafaElZgz** [comment]
Yes, I also tried the command daemon-reload and even a server reboot.

**Comment by 0xJacky** [comment]
This is a bit odd, I just tested changing `/usr/local/etc/nginx-ui/app.ini` on my server and than restart nginx-ui, it works fine.

**Comment by RafaElZgz** [comment]
Idk, later I could try to test this on a fresh ubuntu install.

**Comment by RafaElZgz** [comment]
This is my **app.ini** 
```
[server]
RunMode = release
HttpPort = 9055
HTTPChallengePort = 9180
```
**nginx-ui.service**
```
[Unit]
Description=Yet another WebUI for Nginx
Documentation=https://github.com/0xJacky/nginx-ui
After=network.target

[Service]
Type=simple
ExecStart=/usr/local/bin/nginx-ui -config /usr/local/etc/nginx-ui/app.ini
Restart=on-failure
TimeoutStopSec=5
KillMode=mixed

[Install]
WantedBy=multi-user.target
```
**Execution log**
```
Running in "debug" mode. Switch to "release" mode in production.
run failed: listen tcp :9000: bind: address already in use
```

**Comment by RafaElZgz** [comment]
Same results on other empty Ubuntu 22.04 machine.

**Comment by 0xJacky** [comment]
edit `/usr/local/etc/nginx-ui/app.ini` and then run `systemctl restart nginx-ui`

**Comment by RafaElZgz** [comment]
Well, in the UI settings the port is what I specified: 9055 but the service is running under default port 

![imagen](https://user-images.githubusercontent.com/16884116/213426921-8797733d-62dd-4dfb-94a3-08424a56c7fc.png)

**Comment by RafaElZgz** [comment]
I think there is something bad with the nginx-ui.service

**Comment by 0xJacky** [comment]
The current nginx-ui in your system is already listen on the port you specify and running in background, you do not need to run `./nginx-ui` manually. If you want to run it on the frontground, use `systemctl stop nginx-ui` to kill it, and run `nginx-ui -config /path/to/app.ini`.

**Comment by RafaElZgz** [comment]
No, I specified port 9055 and it is running on port 9000.

**Comment by RafaElZgz** [comment]
The UI is showing what I put on the app.ini but the nginx-ui.service does not use those values.

**Comment by 0xJacky** [comment]
We may need some time to reproduce the issue. Can you please try to use `nginx-ui -config /path/to/app.ini` to run nginx-ui on the frontground, and see which port it is listening on in the console?

**Comment by 0xJacky** [comment]
Sorry @RafaElZgz, that was my mistake, I fixed this in 4c54a9e, and it will be released with the next verison.
